pkg/driver/gid_allocator.go

```diff
@@ -1,84 +1,57 @@
 package driver
 
 import (
-	"container/heap"
-	"sync"
-
+	"context"
+	"fmt"
+	"github.com/kubernetes-sigs/aws-efs-csi-driver/pkg/cloud"
+	"golang.org/x/exp/slices"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
-	"k8s.io/klog"
+	"k8s.io/klog/v2"
+	"sync"
 )
 
-type IntHeap []int
-
-func (h IntHeap) Len() int {
-	return len(h)
-}
-func (h IntHeap) Less(i, j int) bool {
-	return h[i] < h[j]
-}
-func (h IntHeap) Swap(i, j int) {
-	h[i], h[j] = h[j], h[i]
-}
+var ACCESS_POINT_PER_FS_LIMIT int = 120
```

Comment:
This is now 1000: 
https://www.amazonaws.cn/en/new/2023/amazon-efs-increases-the-maximum-number-of-access-points-per-file-system/
https://docs.aws.amazon.com/efs/latest/ug/limits.html